guardian/prnouncer-config
# Use GitHub App authentication instead of PAT

## What does this change?
GitHub Personal Access Tokens are tied to an individual, and thus is a bit fragile. For example, if the individual leaves the organisation, the PAT would become invalid. GitHub Apps are more future proof in that regard.

Inspired by https://github.com/github/combine-prs/blob/main/docs/github-app-setup.md, switch to using a GitHub App to get access to PRs.

The App is named [Gu PRnouncer config](https://github.com/apps/gu-prnouncer-config).

## How to test
[Run](https://github.com/guardian/prnouncer-config/actions/runs/5353332194/jobs/9709107169) one of the Workflows from this branch, and observe messages in Chat:

<img width="843" alt="image" src="https://github.com/guardian/prnouncer-config/assets/836140/4392f552-6ba5-4c69-aa9a-129f81485ee9">
